pallets/circuit/src/lib.rs

```diff
@@ -457,6 +515,17 @@ pub mod pallet {
 
             Ok(())
         }
+
+        fn on_signal(origin: &OriginFor<T>, signal: ExecutionSignal<T::Hash>) -> DispatchResult {
+            log::debug!(target: "runtime::circuit", "Handling on_signal {:?}", signal);
```

Comment:
The rest of logs is already on ::info level, this is fine for RC

Reply:
Hdym? I dont think we wanna promote these to info, since they provide no value to the users